StreamChatUITestsAppUITests/MockServer/User.swift

```diff
@@ -9,13 +9,15 @@ extension StreamMockServer {
 
     func setUpUser(
         source: [String: Any]?,
-        details: [String: String]
+        details: [String: String] = [:]
     ) -> [String: Any]? {
         var user = source?[JSONKey.user] as? [String: Any]
-        user?[UserPayloadsCodingKeys.id.rawValue] = details[UserPayloadsCodingKeys.id.rawValue]
-        user?[UserPayloadsCodingKeys.name.rawValue] = details[UserPayloadsCodingKeys.name.rawValue]
-        user?[UserPayloadsCodingKeys.imageURL.rawValue] = details[UserPayloadsCodingKeys.imageURL.rawValue]
-        user?["image_url"] = details[UserPayloadsCodingKeys.imageURL.rawValue]
+        if !details.isEmpty {
+            user?[UserPayloadsCodingKeys.id.rawValue] = details[UserPayloadsCodingKeys.id.rawValue]
+            user?[UserPayloadsCodingKeys.name.rawValue] = details[UserPayloadsCodingKeys.name.rawValue]
+            user?[UserPayloadsCodingKeys.imageURL.rawValue] = details[UserPayloadsCodingKeys.imageURL.rawValue]
+            user?["image_url"] = details[UserPayloadsCodingKeys.imageURL.rawValue]
```

Comment:
```suggestion
```

Reply:
Hmm, why do you think a user object does not need an `image_url` attribute? 🤔

Reply:
Aren't we assigning it one line above?

Reply:
Haha, not really:
```swift
enum UserPayloadsCodingKeys  {
    case imageURL = "image"
}

Reply:
Removed 👍